taxi/views.py

```diff
@@ -87,3 +89,34 @@ class DriverListView(LoginRequiredMixin, generic.ListView):
 class DriverDetailView(LoginRequiredMixin, generic.DetailView):
     model = Driver
     queryset = Driver.objects.all().prefetch_related("cars__manufacturer")
```

Comment:
When you use `prefetch_related`, it by default returns all objects that satisfy the rule. Therefore, you don't need to explicitly use `objects.all()`